tests/locale/keymap_or_locale.pm

```diff
@@ -37,12 +38,9 @@ sub verify_default_keymap_textmode {
     }
     else {
         send_key('alt-f3');
-        # Make sure the VT switch happened before matching VT content.
-        wait_still_screen;
-        # Some remote backends cannot provide a "not logged in console", so we
-        # also match cleared console. After snapshot rollback we may end up with
-        # cleared console as well.
-        assert_screen([qw(linux-login cleared-console)]);
+        # some remote backends can not provide a "not logged in console" so we
+        # use a cleared remote terminal instead
```

Comment:
I see that to support snapshot rollback I added additional needle tags for the console objects in susedistribution.pm . Should we try something more explicit for the case of rollback, e.g. so that we do not check for boths but for only one of the two to be explicit?

Reply:
To be able to expect only `cleared-console` we need to have the information, that we are after rollback, the snapshot to which we reverted to has cleared console, and that after rollback so far we didn't interacted with the console, so that it got changed. We seems to have none of it.